src/agents/actions/index.ts

```
@@ -38,9 +51,22 @@ async function interpretAction(text: string): Promise<Omit<EmailAction, 'id' | '
       prompt: text,
       schema,
     });
-    return result.object;
+    const obj = result.object;
+    if (obj.command === 'list') {
+      return { command: 'list' };
+    }
+    return {
+      command: 'create',
+      action: obj.action!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)